packages/netlify-cms-widget-select/src/SelectControl.js

```diff
@@ -100,7 +105,8 @@ export default class SelectControl extends React.Component {
       }
 
       return selectedOptions
-        .filter(i => options.find(o => o.value === (i.value || i)))
+        .map(i => options.find(o => o.value === (i.value || i)))
+        .filter(i => i)
```

Comment:
I think you could do `.filter(Boolean)` here instead

Reply:
True. Adapted.